.github/workflows/service_test_gcs.yml

```diff
@@ -39,18 +39,18 @@ concurrency:
 jobs:
   gcs:
     runs-on: ubuntu-latest
-    if: ${{ github.repository_owner == 'apache' }}
+    if: github.event_name == 'push' || !github.event.pull_request.head.repo.fork
```

Comment:
Why are we using `!github.event.pull_request.head.repo.fork`? It seems to me that this disables CI from running if the PR is originated from a forked repository.

Reply:
A fork repo is unable to load credentials.